src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -281,8 +299,12 @@ def values(self) -> ArrayLike:
         >>> idx.values
         array([1, 2, 3])
         """
-        # TODO: SNOW-1458117 implement values
-        return self.to_pandas().values
+        return (
+            self._query_compiler.reset_index()
+            .drop(columns=self._query_compiler.columns)
```

Comment:
Should we move this drop columns to the constructor so we can guarantee all index's df are df with empty data columns?

Reply:
Yep I will do that